### ISSUE TITLE
A real homepage for visitors

### ISSUE THREAD

**Issue by agrodet** [Tatoeba/tatoeba2#2169]
From the things Tatoeba lacks, a _real_ homepage is one of the most important. By _real_, I mean _informative_, _raising curiosity_, _making you want to register_. In that goal in mind, I have set up a first prototype. You can find it below. Here are some notes. Some concern the prototype. Some others are only personal suggestions. 

- Accessing tatoeba.org(/xxx) should redirect to https://tatoeba.org/xxx/about where that homepage should be displayed. Therefore that's not a real homepage but an about page that is a home for visitors. That's a common navigation route, I believe.
- The homepage must insist on a few keywords / ideas. In this prototype, I tried to insist on **help** and **freedom / humanity** (the wording will need special care).
- This prototype is a pure copy of Duolingo homepage. You can have a look at their page for inspiration. 
- Of course, that page should be responsive. One-columned, etc., the usual stuff. 
- Links should be in green, the color of Tatoeba. 
- Obviously, Tatoeba's icon in the middle is not supposed to be cropped.
- Audio should be added in the second block. I didn't add it because if audio is added, a sixth block must be added. I didn't have inspiration to find an interesting sixth block.
- After all that, or after the second block, the **register form could be added** with some motivating wording. "Help us right away!"-like stuff. In my mind, this goes hand in hand with the tutorial / tour for new users that jiru quickly developed in https://github.com/Tatoeba/tatoeba2/issues/2112#issuecomment-586575418. After registration on the homepage, the new user is smoothly moved forward. 

![image](https://user-images.githubusercontent.com/42261966/75608241-30537880-5b41-11ea-8b7b-7c4d8867b304.png)

**Comment by jiru** [comment]
Thank you for your proposal, agrodet. I think the home page for guests has been redesigned several times already, it might be informative to see the previous discussions that led to the current design such as #947.

I agree that the home page for visitors has room for improvement. Let me summarize what I learnt from my [UX tests](https://en.wiki.tatoeba.org/articles/show/ux-tests).
* In test 1, the user went to the home page (using the top-left logo link) in order to learn more about the project, however because she already registered, she was shown the dashboard, so she was disappointed. This suggests we cannot assume that logged-in users will want to see the dashboard instead of the welcome page. I think we need to make these two pages separate: one welcome page and one dashboard page, with the welcome page easily reachable from the dashboard page.
* In test 1, the user wanted to know specifically about the philosophy, the history and the motivations of the project, especially the preservation of endangered languages (which I all mentioned to her before the test) and the fact that Tatoeba is an NPO (which she found really cool).
* In test 1, the home page and the search bar force the user to discover Tatoeba through a search form while she is not interested in searching yet. Instead, the user wanted to:
  * explore the list of languages (she did that by scrolling the language dropdown)
  * explore language pairs (I needed to explain her to leave the keywords field blank)
  * explore sentences of a given language (I needed to explain her to set the language dropdown to "Any language")
* In test 3, the user was disappointed by seeing English mixed with Japanese on the homepage.
* In test 3, after the test, the user explained that if she were to introduce Tatoeba to her students, she wouldn’t know how to explain in which way it could be useful to them. I think this something a real home page should be able to answer. The user also praised that Tatoeba includes contents contributed by native speakers (unlike what you can find in textbooks). This is probably a relevant information to put on the home page.

**Comment by agrodet** [comment]
Thanks for the summary! 

> In test 3, the user was disappointed by seeing English mixed with Japanese on the homepage.  

Could you remind me what that problem was exactly, or what part of your UX test it refers to?

> the philosophy, the history and the motivations of the project, especially the preservation of endangered languages (which I all mentioned to her before the test) and the fact that Tatoeba is an NPO (which she found really cool).

I think, then, that what's on the current About page isn't enough. Are these information given somewhere? If somebody could give me some rough sketch about what to say concerning these points (maybe @trang ?), I could prepare Prototype 2. 

I've been thinking about Prototype 2 and it should improve the situation about most of the points you gave, except the last one (that's a tough part, I think).

**Comment by trang** [comment]
> If somebody could give me some rough sketch about what to say concerning these points

I cannot give you a rough sketch but I can tell you where to get the necessary information to figure out of what to say concerning these points.

For the philosophy and motivations of the project, I think you can extract some key ideas from the video on the [About](https://tatoeba.org/eng/about) page. Probably more than 90% of the content of this video is still relevant today.

For the history, I once wrote a [blog article about it](https://blog.tatoeba.org/2013/05/the-story-of-tatoeba.html). At least if you want to know a bit more about the very early days, how it all started.

Preservation of endangered languages was not exactly part of the initial vision, but as the project grew, we realized that Tatoeba could help on that front. We haven't been actively trying to preserve endangered languages, in the sense that we haven't been specifically searching for native speakers of endangered languages to present them Tatoeba. It's just that at some point we decided that we're going to support every language in the world and that includes endangered languages.

Aside of that, I hope we can put a bit more emphasis on the fact that Tatoeba is open source and I hope we can put it in a way that is inviting to people who are just starting to learn how to code, or don't know much about programming yet but would like to learn.

**Comment by ckjpn** [comment]
> Links should be in green, the color of Tatoeba.

As someone who is aware of problem people with color-blindness have, I'd like to see this low-contrast color green link color replaced with something else sitewide.

It's possible to compensate for this problem with Stylish people's personal computers, but the problem reappears whenever using other people's computers or mobile devices.

Related issue:  https://github.com/Tatoeba/tatoeba2/issues/1164

**Comment by ckjpn** [comment]
>  I think we need to make these two pages separate: one welcome page and one dashboard page, with the welcome page easily reachable from the dashboard page.

Perhaps having one home page with appropriate links that is shared by everyone would be a good idea.

Members could opt-in to the "dashboard page" in their settings to be redirected to it from http://tatoeba.org.  Or, you could have the setting to automatically go to the dashboard only after a member has contributed a sentence, but still have a setting that would let members choose which way they preferred.  Also, have a link from the dashboard page to easily allow members to get back to the main home page.


## Example of a Possibly Useful Home Page for Everyone.

Have a simple page with the following links.

- **About the Project** Having a very brief one-or-two line explanation with a link to **more...** would likely be best.

- **Example of what we have.** Have a link that a visitor can click to show a random selection of sentences, possibly limited to only ones by native speakers or some other method of giving new members a positive feeling about the project.
-- Possibly have 3 or 4 different links instead of just one (Show a Random Selection of Languages. Show English with Audio Linked to Other Sentences with Audio, Show Sentences tagged as "imperative", etc.  The purpose wouldn't be to give visitors an exhaustive list of possibilities, but to let them know what some of the possibilities are.).
-- Note that rather than just one randomly-selected sentence like we have now, allowing visitors to show multiple randomly-selected sentences will give them a better idea of what the site is about, and also avoids the problem of having one pornographic sentence or obviously incorrect sentence being shown, which can easily affect a visitor's first impression of the project.

- **Search our database.**  I'm not sure, but perhaps having a page set up specifically for searching might be better than to have the search form on the main home page.

- **How you can help.** Direct the visitor to a page about registering and how they can add translations, etc.

- **Other things you can do.** Direct the visitor to a page explaining lists, etc.

- **See Activity** Direct the visitor to the currently-used home page for logged in members.

I'm not necessarily recommending the above wording for the links. I think that with a little brainstorming that we can come up with concise, descriptive ways to say some of the above.

**Comment by jiru** [comment]
@agrodet 
>> In test 3, the user was disappointed by seeing English mixed with Japanese on the homepage.
>
> Could you remind me what that problem was exactly, or what part of your UX test it refers to?

It’s at the very beginning of test 3 (homepage for guests). The user is disappointed because she realizes that only part of the website is in Japanese while the rest is in English, but she doesn’t understand English. It happens later in the test too, at the section `Tatoeba wiki page "Contribute audio for Tatoeba"` where she runs away from a full-English page. I think it’s understandable: it’s not great to have to use a website in a language that you cannot understand at all. Apart for working better on the UI translations, I think we could put a banner or something to inform/apologize about the partial translation.

**Comment by agrodet** [comment]
Simple prototype 2. The width of the whiteframe could / should be increased. The stats block at the bottom will of course be modified to the one we have now, showing the Activity Timeline link. I'm not satisfied with the video being in the same block as "Built by humans" but since I moved some text to the green header part, the balance was not correct. Maybe prototype 3 will change that. 

Opinions and new inputs are welcome.

![image](https://user-images.githubusercontent.com/42261966/80494209-60809180-89a1-11ea-85a5-a3c5eca5949b.png)

**Comment by jiru** [comment]
It’s really cool that you are working on this, @agrodet! I like your home page but I think there is room for improvement. My very first impression is: too much text, too long.

In particular, the green banner immediately attracts attention, however, unlike the "Build by humans" text, its content is not very well structured (and contain unnecessary details like the origin of the name Tatoeba). Since it’s the very first thing you see, I suggest making it shorter and simpler. A slogan-like phrasing like in the current home page is good enough.

Speaking of which, the current green banner (*Tatoeba is a collection of sentences and translations. It's collaborative, open, free and even addictive.*), while correct, have always bugged me because it doesn’t tell much what Tatoeba is all about. I imagine reading this for the first time and asking myself: so what? What can I do with that? It’s like describing a dictionary project by saying "it’s a large collection of words". I think using words like "dictionary" or "corpus" could help making sense. On the other hand, such words narrow the project to a single use: dictionary → language learning; corpus → research. The fact that Tatoeba sticks on the "data" level without pushing for a particular use of that data makes it harder to promote because the use case is unclear. This issue has been raised in the [debrief of UX test 3](https://en.wiki.tatoeba.org/articles/show/ux-test-3#debrief) by the way.

How about using an image like the [image of interconnected languages in the video](https://youtu.be/ac9SmJuwHqk?t=88)? A good image worth a thousand words, they say…

I like the phrase "build by humans", however I don’t think it fits as a title for the subsections. The section about vocabulary collection should be rephrased into vocabulary request (see #2277).

I like the idea of showing random sentence pairs instead of just a random sentence, but the form feels like too much hassle to fill in. Here is a suggestion:
* use a more simple phrasing like "Show sentence pairs:" [from] <=> [to]
* prefill [from] with the UI language and [to] with a random language so that *a sentence pair is already showed*, and the user may just have to change the [to] to see something interesting

Maybe the section "Search for the words you want in the language you want" could be advantageously merged into the top search form.

I think a missing information from your proposed home page is that we support *a lot* of languages. In [UX test 1](https://en.wiki.tatoeba.org/articles/show/ux-test-1#exploration), the user took quite some time to explore the list of languages (by scrolling the language drop down, which is not the best way to explore the list). I don’t know what’s the best way to convey that information, but since I’m working on #2294, you could insert a link to that new page.

Another important missing information, in my opinion, is the community aspect of Tatoeba. I mean meeting other people interested in languages, exchanging with people from different cultures, correcting each other, asking questions, learning new things, asking for vocabulary, etc. It is true that the communication spaces and tools are quite limited on Tatoeba, and the social aspect of the project has never been much valuated, but I think it is quite of a key factor that make people stick. I mean what Tatoeba would be good for without all the cool people I met there? If you look at UX test 1 that I linked above, the first thing the user tried to do is to find other members speaking her language…

**Comment by ckjpn** [comment]
> ... (Tatoeba is a collection of sentences and translations. It's collaborative, open, free and even addictive.), while correct, have always bugged me because it doesn’t tell much what Tatoeba is all about. 

Perhaps aiming at something descriptive like this would work.

Tatoeba.org is a large database of example sentences translated into many languages by its members who volunteer their time.

**Comment by agrodet** [comment]
Maybe we should think of a slogan for Tatoeba. Or a catchy sentence. "Lots of languages, lots of sentences, lots of love" but in a non-stupid, catchy way :)

**Comment by trang** [comment]
For the record, here's the Wall thread that led to our current short description of Tatoeba:
https://tatoeba.org/eng/wall/show_message/21969#message_21969

**Comment by ckjpn** [comment]
https://prnt.sc/6beuq6 in that Wall post no longer exists
Perhaps you could post the image here in this GitHub issue.

**Comment by trang** [comment]
I don't have this image.
